alacritty.yml

```diff
@@ -5,14 +5,14 @@
 # environment variables. Some entries may override variables
 # set by alacritty itself.
 env:
-  # TERM env customization
+  # TERM variable
   #
-  # If this property is not set, alacritty will set it to `xterm-256color`.
+  # This value is used to set the `$TERM` environment variable for
+  # each instance of Alacritty. If it is not present, alacritty will
+  # check the local terminfo database and use 'alacritty' if it is
+  # available, otherwise 'xterm-256color'
```

Comment:
This is such a nice sentence, it's a shame to end it so abruptly. I'd add a `is used.` at the end to round it off a bit nicer.